packages/ember-toucan-core/src/components/form/controls/file-input.hbs

```diff
@@ -1,4 +1,5 @@
 <input
+  accept={{this.accept}}
```

Comment:
:+1:

Reply:
Thanks for adding this!

Reply:
I wondered about this as well, but then found that this [already exists](https://github.com/CrowdStrike/ember-toucan-core/blob/64265d5bc77e2426c0152f9177bc4089f786f196/packages/ember-toucan-core/src/components/form/controls/file-input.ts#L38-L40), probably was just not used before, or lost during some refactoring/rebasing, idk?

Reply:
Yup! That's what I was thinking.  Maybe lost during refactoring or rebasing.  It is in the backing JS file, but it wasn't actually used anywhere (until now!)

Reply:
Ahh, thanks for linking me to it!